app/src/main/java/org/simple/clinic/patient/sync/PatientSync.kt

```diff
@@ -73,7 +74,9 @@ class PatientSync @Inject constructor(
                 address = address.toPayload(),
                 phoneNumbers = numberPayloads,
                 businessIds = businessIdPayloads,
-                recordedAt = recordedAt)
+                recordedAt = recordedAt,
+                reminderConsent = Granted
```

Comment:
Shouldn't this be reading the `reminderConsent` from the `Patient` instead of hardcoding it to `Granted`?

Reply:
Right! Will fix.